Scripts/Runtime/Core/DOTweenActions/DOTweenActionBase.cs

```diff
@@ -26,6 +26,8 @@ public enum AnimationDirection
         protected CustomEase ease = CustomEase.InOutCirc;
         [SerializeField]
         protected bool isRelative;
+        [SerializeField]
+        protected bool canRewind = true;
```

Comment:
maybe we should expose the `autoKill` option here, would make more sense don't you think?

Reply:
you're absolutely right, that makes much more sense! pushed new version